pkgs/development/libraries/libtickit/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, libtermkey
+, unibilium
+, libtool
+}:
+stdenv.mkDerivation rec {
+  pname = "libtickit";
+  version = "0.4.3";
+
+  src = fetchFromGitHub {
+    owner = "leonerd";
+    repo = "libtickit";
+    rev = "v${version}";
+    hash = "sha256-QCrym8g5J1qwsFpU/PB8zZIWdM3YzOySknISSbQE4Sc=";
+  };
+
+  makeFlags = [
+    "PREFIX=$(out)"
+    "LIBTOOL=${lib.getExe libtool}"
+  ];
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ libtermkey unibilium libtool ];
```

Yeah, reading https://github.com/NixOS/nixpkgs/pull/238657#discussion_r1248062132, it should be

```suggestion
  nativeBuildInputs = [
    pkg-config
    libtool
  ];

  buildInputs = [
    libtermkey
    unibilium
  ];
```

---

I moved `libtool` to `nativeBuildInputs` as of 933ec09c1724b0c7fb5d0b4bdfbe921a1920391b. Should I also format the dependencies like the commit suggestion?